src/app/App.scss

```diff
@@ -205,4 +205,9 @@
   border-right: 1px solid var(--pf-c-select__toggle--before--BorderRightColor);
   border-bottom: 1px solid var(--pf-c-select__toggle--before--BorderBottomColor);
   border-left: 1px solid var(--pf-c-select__toggle--before--BorderLeftColor);
-}
\ No newline at end of file
+}
+
+// centers modals when a quick start drawer is open
+.pf-c-drawer.pf-m-expanded .pfext-quick-start-drawer__body .pf-c-backdrop {
+  position: absolute;
```

Comment:
Looks like there is a variable here we can use `--pf-c-backdrop--Position: absolute;`
and, possibly the selector can just be `.pfext-quick-start-drawer__body .pf-c-backdrop`? 

Inside kafka, I see the modals overlay the entire screen including quickstart. Is that preferable and possible? 
![image](https://user-images.githubusercontent.com/19825616/140321221-42a9948d-c657-404f-9ada-07e59cf7dd6a.png)

Reply:
The goal is to keep the quick start drawer on the right in view and not blocked out by the modal. That is so a user can follow instructions that might take them through modals. For that to happen, 2 things need to be done:
1. modals append themselves to the `#chrome-app-render-root`
2. the backdrop needs to be absolute and not fixed

I've made the first change to some of the modals, like the 'Create kafka instance' modal.
<img width="1680" alt="Screen Shot 2021-11-04 at 1 50 07 PM" src="https://user-images.githubusercontent.com/869106/140393584-e9f48438-31f8-4c3c-9e26-7cae99385816.png">

Changing the backdrop position to absolute will center the modal:
<img width="1680" alt="Screen Shot 2021-11-04 at 1 50 49 PM" src="https://user-images.githubusercontent.com/869106/140393727-e6cb6dcc-d83f-4d8a-b0e1-a6f8c7816b8e.png">

I can see if the `--pf-c-backdrop--Position` variable can be changed to do that instead though!

Reply:
Sounds good :) And try reducing the selector; that was a very high specificity and I think you can get away with just scoping it inside the quickstart drawer body but I'm not positive.